spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoEmulatorProperties.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.datastore;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties for configuring Cloud Datastore Emulator.
+ *
+ * @author lucasoares
+ *
+ * @since 1.2
+ */
+@ConfigurationProperties("spring.cloud.gcp.datastore.auto-emulator")
```

Comment:
In my first try I did that, but became a little odd since the emulator-host property exists.. I replied your other comment and I will wait some advise to the possibility of changing emulator-host to only host.

Reply:
Let's deprecate `emulator-host` and replace it with `host`.

Reply:
Ok then.. But I still have a question but we talk about this in the other conversation. Closing this one.